packages/suite-desktop-core/e2e/tests/analytics/events.test.ts

```diff
@@ -124,12 +127,16 @@ test.describe('Analytics Events', { tag: ['@group=suite', '@webOnly'] }, () => {
             // the only message about the analytics being sent is the "settings/analytics" disabled.
             await analytics.interceptAnalytics();
 
-            await trezorUserEnvLink.startEmu({ wipe: true, model: 'T3T1' });
+            await trezorUserEnvLink.startEmu({ wipe: true, model: emulatorStartConf.model });
```

Comment:
we could do 
```suggestion
            await trezorUserEnvLink.startEmu({ wipe: true, model: model.model });
```
and drop  the emulatorStartConf

Reply:
done